e2e-jar-test/core/src/main/java/net/snowflake/IngestTestUtils.java

```diff
@@ -161,6 +168,30 @@ public void test() throws InterruptedException {
     waitForOffset(channel, offset);
   }
 
+  public void runLongRunningTest(Duration testDuration) throws InterruptedException {
```

Comment:
This will be part of the merge gate? This is too long to run and we should run it outside of the merge gate (like what we did for micro-benchmark)

Reply:
This will not be a part of precommit, I will add `@Ignore` before I merge.